docs/user-guide/featured.md

```diff
@@ -0,0 +1,29 @@
+## Featured Notebooks & Collections
```

Comment:
Any time you create a new markdown file you should add it to `toctree` in `index.rst` to make it easy for users to navigate.
You can observe that the same order is been used to render the left pane in our docs page.
![image](https://user-images.githubusercontent.com/24864161/97275275-be603a00-185b-11eb-867d-abef8c4a1c1d.png)

You can add `user-guide/featured` after `user-guide/topics`